### PR TITLE
#15224 add warning if failed to parse Date

### PR DESCRIPTION
**Pull request by Destrolaric**

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/internal/ResultSetMessages.properties [hunk 1]
```diff
@@ -156,6 +156,9 @@ dialog_value_view_dialog_error_updating_title = Error updating column
 dialog_value_view_job_selector_name = Select 
 dialog_value_view_label_dictionary = Dictionary  (<a>{0}</a>): 
 
+dialog_value_view_error_parsing_date_title = Error parsing date
+dialog_value_view_error_parsing_date_message = Calendar is not able to parse this value!
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/internal/ResultSetMessages.properties]:
```suggestion
dialog_value_view_error_parsing_date_message = Calendar is not able to parse this value
```

**Comment by serge-rider** [comment]
1. Do not use exlamation marks in messages
2. Show wrong date/tiem literal in the error message

File: plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/internal/ResultSetMessages.properties [hunk 2]
```diff
@@ -157,7 +157,7 @@ dialog_value_view_job_selector_name = Select
 dialog_value_view_label_dictionary = Dictionary  (<a>{0}</a>): 
 
 dialog_value_view_error_parsing_date_title = Error parsing date
-dialog_value_view_error_parsing_date_message = Calendar is not able to parse this value!
+dialog_value_view_error_parsing_date_message = Calendar is not able to parse this value {0}
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/internal/ResultSetMessages.properties:160]:
```suggestion
dialog_value_view_error_parsing_date_message = Unable to parse value {0}
```

File: plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/data/editors/DateTimeInlineEditor.java [hunk 3]
```diff
@@ -120,8 +121,9 @@ public void run() {
                 }
             }
             try {
-                if (!(parent.extractEditorValue() instanceof Date)){
-                    UIUtils.asyncExec(() -> DBWorkbench.getPlatformUI().showWarningMessageBox(ResultSetMessages.dialog_value_view_error_parsing_date_title, ResultSetMessages.dialog_value_view_error_parsing_date_message));
+                Object value = parent.extractEditorValue();
+                if (!(value instanceof Date)){
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/data/editors/DateTimeInlineEditor.java:124]:
```suggestion
                if (!(value instanceof Date)) {
```

File: plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/data/editors/DateTimeInlineEditor.java [hunk 4]
```diff
@@ -120,8 +121,9 @@ public void run() {
...
+                    UIUtils.asyncExec(() -> DBWorkbench.getPlatformUI().showWarningMessageBox(ResultSetMessages.dialog_value_view_error_parsing_date_title, NLS.bind(ResultSetMessages.dialog_value_view_error_parsing_date_message, value)));
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/data/editors/DateTimeInlineEditor.java]: This should be an error, not a warning. See https://github.com/dbeaver/dbeaver/pull/15232#issuecomment-1020470245

**Review by ShadelessFox — CHANGES_REQUESTED** [review]

**Pull request merged**